lib/baw-workers/file_info.rb

```diff
@@ -183,5 +185,20 @@ def file_name_datetime(file_name, utc_offset = nil)
       result
     end
 
+    # Convert a .wac file to a .wav file in the same directory.
+    # @param [String] file_path
+    # @return [String] converted file path
+    def convert_wac_to_wav(file_path)
+      ext = File.extname(file_path)
+      fail BawAudioTools::Exceptions::AudioToolError, 'Can only convert from .wac' if ext != '.wac'
+
+      source = File.expand_path(file_path)
+      target = File.join(File.dirname(source), File.basename(source, '.wac') + '.wav')
+
+      # only try to create the wav file if it doesn't already exist
+      @audio.modify(source, target) unless File.size?(target)
```

Comment:
@cofiem - why this `unless` condition? Couldn't this lead to non-idempotent behaviour on repeated runs?

I guess you're assuming an identical .wav file may have already been uploaded and you don't want to run wac2wav if that is the case... in which case, that check should be explicit.

Adding to that, the output name of the .wav file should be unique so duplicate runs don't mistake converted wav files as original wav files.